src/components/common/Buttons/VerticalKebabDrop/VerticalKebabDrop.jsx

```diff
@@ -0,0 +1,23 @@
+import { DropdownBtn } from '@/components/common/Buttons';
+import kebabIcon from '@/assets/icons/vertical_kebab.svg';
+import styles from './VerticalKebabDrop.module.scss';
+
+const VerticalKebabDrop = ({ onSelect }) => {
+  const options = [
+    { label: '수정하기', value: 'edit' },
+    { label: '삭제하기', value: 'delete' },
+  ];
```

Comment:
💊 제안
common 폴더에 있는 만큼, options를 외부에서 받아 다른 메뉴일 때도 사용할 수 있게 설계하시면 좋겠어요!
```suggestion
const VerticalKebabDrop = ({ options, onSelect }) => {
```